app/assets/javascripts/discourse/tests/unit/lib/get-url-test.js

```diff
@@ -26,7 +26,7 @@ module("Unit | Utility | get-url", function () {
     setPrefix("/eviltrout");
     assert.equal(withoutPrefix("/eviltrout/hello"), "/hello");
     assert.equal(withoutPrefix("/eviltrout/"), "/");
-    assert.equal(withoutPrefix("/eviltrout"), "");
+    assert.equal(withoutPrefix("/eviltrout"), "/");
```

Comment:
This is a breaking change. Is there any way to fix it and maintaining reverse compatibility with the old API?

Reply:
Yes, sure, I've changed it to avoid this breaking change.